rs/ledger_suite/icrc1/ledger/canbench_results/canbench_u64.yml

```diff
@@ -1,42 +1,42 @@
 benches:
   bench_icrc1_transfers:
     total:
-      instructions: 5926738616
-      heap_increase: 214
+      instructions: 13237283790
+      heap_increase: 171
       stable_memory_increase: 128
     scopes:
       before_upgrade:
-        instructions: 5341274673
-        heap_increase: 64
+        instructions: 13068913917
```

Comment:
`before_upgrade` represents 10k transfers and with balances in stable memory the number of instructions increases by ~130%. The upgrade (below) instruction and heap usage decreased, as expected.

Reply:
Maybe this test is too small to have big impact on stable memory. I ran a test with 100k, 200k and 300k balances and I saw increases of 250-300 stable memory pages per 100k balances.